package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "xmlhttprequest-ssl",
   "description": "XMLHttpRequest for Node",
-  "version": "1.5.5",
+  "version": "1.5.6",
```

Comment:
In my opinion your changes will at least warrant a minor version bump. Technically it should even be a major, but I doubt anyone's really relying on the specific error type or 503 status code being returned. If you could change it to be `1.6.0` that'd be awesome!